### PR TITLE
add docs: failover policy overview

### PR DESCRIPTION
**Pull request by mrlihanbo**
Signed-off-by: lihanbo <lihanbo2@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add docs about the failover policy in karmada

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
"NONE"

### CODE REVIEW

File: docs/userguide/failover.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+# Failover Policy Overview
```
Comment (RainbowMango) [inline docs/userguide/failover.md:1]:
```suggestion
# Failover Overview
```

**Comment by RainbowMango** [comment]
```diff
diff --git a/docs/README.md b/docs/README.md
index 72e2bfbf..15d77c07 100644
--- a/docs/README.md
+++ b/docs/README.md
@@ -13,6 +13,7 @@ Refer to [Installing Karmada](./installation/installation.md).

 - [Cluster Registration](./userguide/cluster-registration.md)
 - [Aggregated Kubernetes API Endpoint](./userguide/aggregated-api-endpoint.md)
+- [Cluster Failover](./userguide/failover.md)

```

File: docs/userguide/failover.md (hunk 1 above)
Comment (mrlihanbo) [reply docs/userguide/failover.md]: done

**Comment by mrlihanbo** [comment]
> ```diff
> diff --git a/docs/README.md b/docs/README.md
> index 72e2bfbf..15d77c07 100644
> --- a/docs/README.md
> +++ b/docs/README.md
> @@ -13,6 +13,7 @@ Refer to [Installing Karmada](./installation/installation.md).
> 
>  - [Cluster Registration](./userguide/cluster-registration.md)
>  - [Aggregated Kubernetes API Endpoint](./userguide/aggregated-api-endpoint.md)
> +- [Cluster Failover](./userguide/failover.md)
> ```

done

**Review by RainbowMango — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**